dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -139,20 +158,18 @@ public void testQuery() throws Exception {
 
     @Test
     public void testQueryWithDifferentAssembly() throws Exception {
-        SubSnpCoreFieldsReader fieldsReader = buildReader("Bos_taurus_UMD_3.1.1",
-                                                          Collections.singletonList("Primary_Assembly"), 2000);
-        List<SubSnpCoreFields> list = readAll(fieldsReader);
-        assertEquals(2, list.size());
-        for (SubSnpCoreFields subSnpCoreFields : list) {
-            assertNotNull(subSnpCoreFields);
-            assertTrue(expectedRsIds.containsKey(subSnpCoreFields.getRsId()));
-        }
+        reader = buildReader(CHICKEN_ASSEMBLY_4, Collections.singletonList(PRIMARY_ASSEMBLY), PAGE_SIZE);
+        List<SubSnpCoreFields> list = readAll(reader);
+
+        assertEquals(1, list.size());
+        assertEquals(snpInDifferentAssembly, list.get(0));
     }
 
     @Test
     public void testQueryWithDifferentAssemblyType() throws Exception {
-        SubSnpCoreFieldsReader fieldsReader = buildReader("Btau_5.0.1", Collections.singletonList("non-nuclear"), 2000);
-        List<SubSnpCoreFields> list = readAll(fieldsReader);
+        reader = buildReader(CHICKEN_ASSEMBLY_5, Collections.singletonList("non-nuclear"), PAGE_SIZE);
```

Comment:
"non-nuclear" would also be a good constant, because it is one of the usual assembly types along with "Primary_assembly".